.gitignore

```diff
@@ -2,12 +2,12 @@
 
 # compiled output
 /dist
```

Comment:
this can all be shortened to
```
dist/
node_modules/
tmp/
```

the lack of leading `/` means that any subdir will match

Reply:
should be done